test/ngrok.guest.spec.js

```diff
@@ -165,5 +165,9 @@ describe('guest.spec.js - ensuring no authtoken set', function() {
 			});
 
 		});
+
+		after(async () => {
```

Comment:
I needed to add this - the postinstall integration tests would fail because they were trying to write the binary out.